README.md

```diff
@@ -1,85 +1,126 @@
 ```
 
 ## Configuration
-Here is a description with the values that the mitreid and ssp conf file must include
+
+Here is a description with the values that the MITREid and SSP conf file must include
 example_deployers.config.json
-```javascript
+
+```json
 {
-  "ssp": {
+  "keycloak": {
     "ams": {
       "host": "example.host.com",
-      "project" : "ams-project-name-ssp",
-      "pull_topic" : "ams-topic-ssp",
-      "pull_sub" : "ams-sub-ssp",
-      "token" : "ams-token-ssp",
-      "pub_topic": "ams-publish-topic-ssp",
+      "project": "ams-project-name-mitreid",
+      "pull_topic": "ams-topic-mitreid",
+      "pull_sub": "ams-sub-mitreid",
+      "token": "ams-token-mitreid",
+      "pub_topic": "ams-publish-topic-mitreid",
       "poll_interval": 1,
       "agent_id": "1"
     },
-    "metadata_conf_file": "/path/to/ssp/metadata/file.php",
-    "cron_secret": "SSP cron secret",
-    "cron_url": "http://localhost/proxy/module.php/cron/cron.php",
-    "cron_tag": "hourly",
-    "request_timeout": 100
+    "issuer": "https://example.com/oidc",
+    "refresh_token": "refresh token",
+    "client_id": "client ID",
+    "client_secret": "client secret",
+    "default_client_scopes": []
   },
   "mitreid": {
     "ams": {
       "host": "example.host.com",
-      "project" : "ams-project-name-mitreid",
-      "pull_topic" : "ams-topic-mitreid",
-      "pull_sub" : "ams-sub-mitreid",
-      "token" : "ams-token-mitreid",
+      "project": "ams-project-name-mitreid",
+      "pull_topic": "ams-topic-mitreid",
+      "pull_sub": "ams-sub-mitreid",
+      "token": "ams-token-mitreid",
       "pub_topic": "ams-publish-topic-mitreid",
       "poll_interval": 1,
       "agent_id": "1"
     },
-    "issuer": "https://example.com/oidc" ,
+    "issuer": "https://example.com/oidc",
     "refresh_token": "refresh token",
     "client_id": "client ID",
     "client_secret": "client secret"
   },
+  "ssp": {
+    "ams": {
+      "host": "example.host.com",
+      "project": "ams-project-name-ssp",
+      "pull_topic": "ams-topic-ssp",
+      "pull_sub": "ams-sub-ssp",
+      "token": "ams-token-ssp",
+      "pub_topic": "ams-publish-topic-ssp",
+      "poll_interval": 1,
+      "agent_id": "1"
+    },
+    "metadata_conf_file": "/path/to/ssp/metadata/file.php",
+    "cron_secret": "SSP cron secret",
+    "cron_url": "http://localhost/proxy/module.php/cron/cron.php",
+    "cron_tag": "hourly",
+    "request_timeout": 100
+  },
   "log_conf": "conf/logger.conf"
 }
 ```
 
-As shown above there are 2 main groups mitreid and ssp, for each group there are unique ams settings and service specific configuration values. The only global value is the `log_conf` path if you want to use the same logging configuration for both of the deployers. In case you need a different configuration for a deployer you can add log_conf in the scope of "mitreid" or "ssp"
+As shown above there are 2 main groups MITREid and SSP, for each group there are unique AMS settings and service
```

Comment:
As shown above there are three main groups, namely `keycloak`, `mitreid` and `ssp` and each group can have its own AMS settings and service